_data/sites.yml

```diff
@@ -2186,12 +2186,17 @@
   url: https://pawelgrzybek.com/
   size: 29.1
   last_checked: 2022-03-18
-
+  
```

Comment:
Thanks for adding your site! Do you mind removing this extra whitespace?

Reply:
no problem